<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataManageUI</class>
 <widget class="QWidget" name="dataManageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>方正启体简体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>OpenCV Face Recognition System - DataManage</string>
  </property>
  <widget class="QGroupBox" name="dbManageGroupBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>341</width>
     <height>661</height>
    </rect>
   </property>
   <property name="title">
    <string>数据管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="trainGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>训练数据</string>
    </property>
    <widget class="QPushButton" name="trainButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>数据库未完成初始化，请检查</string>
     </property>
     <property name="text">
      <string>开始训练</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="deleteUserGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>删除用户</string>
    </property>
    <widget class="QGroupBox" name="queryUserGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string> 学号查询</string>
     </property>
     <widget class="QLabel" name="queryResultLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLineEdit" name="queryUserLineEdit">
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="placeholderText">
          <string>请输入8位学号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryUserButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>数据库未完成初始化，请检查</string>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="queryResultGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>查询结果</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>98</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="infoHorizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="keyVerticalLayout">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="stuIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>学号：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cnNameLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="faceIDLabel">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Face ID：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="valueVerticalLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLineEdit" name="stuIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="faceIDLineEdit">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="deleteUserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>删除记录</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <widget class="QTextEdit" name="logTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>321</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="faceDbGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>661</height>
    </rect>
   </property>
   <property name="title">
    <string>人脸库</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="dbVisualGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>721</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>591</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Face ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>注册时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>班级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>手机号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>地址</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>590</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注：Face ID为 -1 说明该用户的人脸数据未被训练</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
